sickbeard/showUpdater.py

```diff
@@ -105,7 +105,10 @@ def run(self, force=False):  # pylint: disable=unused-argument, too-many-locals,
             try:
                 if indexer_name == 'theTVDB':
                     if cur_show.indexerid in updated_shows:
-                        pi_list.append(sickbeard.showQueueScheduler.action.updateShow(cur_show, True))
+                        if not cur_show.paused:
+                            pi_list.append(sickbeard.showQueueScheduler.action.updateShow(cur_show, True))
+                        else:
+                            logger.log(u"Show update skipped, show: %s is paused." % cur_show.name, logger.INFO)
```

Comment:
We're trying to use .format() everywhere since a few months. And just cleanup when touching something. For showUpdater.py we should replace the % (). As now where mixing them.

tv.py is a different case, as it's too big, to do it in a small PR.